ml-agents.Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM nvidia/cuda:10.0-cudnn7-devel-ubuntu18.04
+LABEL maintainer=ml-agents@unity3d.com
+
```

Comment:
Add:
```
ENV LANG C.UTF-8
```
so everybody knows we use UTF-8 encoding.
```
# Set the environment to noninteractive (no user input)
ENV DEBIAN_FRONTEND noninteractive
```
so all the installers known for sure we can not interact with them.